Dockerfile

```diff
@@ -1,28 +1,23 @@
-FROM docker-registry.eyeosbcn.com/eyeos-fedora21-node-base:latest
+FROM docker-registry.eyeosbcn.com/alpine6-node-base
 MAINTAINER eyeos
 
 ENV WHATAMI ldap
 ENV LDAP_LOG_LEVEL 0
 
-COPY files/monitor /tmp/
-COPY files/hdb /tmp/
 COPY files/base.ldif /tmp/
```

Comment:
Please combine all these COPY statements into one command. It will further reduce the size of the docker image.